test/benchmarks/CMakeLists.txt

```diff
@@ -3,6 +3,9 @@ cmake_minimum_required(VERSION 3.12)
 include(FetchContent)
 
 enable_language(CXX)
+set(CMAKE_CXX_STANDARD 11)          # The CXX standard required to build this target
```

Comment:
I am not so sure the comments are necessary.

I am wondering if these should be set/enabled before the `enable_language` call.

Reply:
It fixed my compilation problems as it was, but I changed the order anyways to avoid any possible confusion.